app/services/claims/fee_calculator/unit_price.rb

```diff
@@ -0,0 +1,85 @@
+# Service to calculate the unit price for a given fee.
+# Unit price will require input from different attributes
+# on the claim and may require input from different CCCD fees
+# to be consolidated/munged.
+#
+module Claims
+  module FeeCalculator
+    # TODO: this is using:
+    # 1. for primary/main fixed fees it use the calculate endpoint with quantity/days of 1 and no modifiers
+    # 2. for case uplifts it calculates for the matching "primary" fee type with number of cases
+    #    modifier of 2 (1 additional case) and "days/quantity" taken from the primary fee types' quantity.
+    # However, could use the prices endpoint of API directly or request a new endpoint to reduce number
+    # of calls and/or simplify.
+    #
+    class UnitPrice < Calculate
+      def call
```

Comment:
**[DRY]** This call method seem to be exactly the same as its parent class with the exception of the locale for the error message.